index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1>My logo</h1>
+    <div id="hamburgerContainer">
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
+    </div>
+  </header>
+  <main>
+    <section id="intro">
+      <h2>
+        Hey There.
+        I’m Yuliana.
+      </h2>
```

Comment:
- [ ] As it stands as the main headline, kindly make it a `h1` tag.
- If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

Reply:
Hello @Shedrack-Sunday, thanks for your comments.
I already used the H1 tag in the header as it is a common practice to use the logo or the initial part of the header as an H1, and according to best practices I should have only one H1 on the entire page, should I remove the H1 from the header or what is the best procedure in this case?
Regards